### ISSUE TITLE
$count=true&$top=0 don't work

### ISSUE THREAD

**Issue by RrOoSsSsOo** [flat3/lodata#518]
The combination of `$count=true&$top=0` to return only count (and a void set of values), like Odata behaviour, seems not work.

The example [https://services.odata.org/V4/Northwind/Northwind.svc/Customers?$count=true&$top=0](https://services.odata.org/V4/Northwind/Northwind.svc/Customers?$count=true&$top=0) returs that there are 91 results between Customers where the Country is 'Germany', but with an empy set of value `[]`)
```
{
  "@odata.context": "https://services.odata.org/V4/Northwind/Northwind.svc/$metadata#Customers",
  "@odata.count": 91,
  "value": []
}
```

Lodata fill the value with results...
```
{
    "@context": "http://localhost:80/odata/$metadata#Customers",
    "value": [
        {
            ...
        },
        ... ,
        {
            ...
        }
    ],
    "@count": 3511,
    "@nextLink": "http://localhost:80/odata/Customers?%24count=true&top=200&skip=200"
}
```

(Other resources from [stackoverflow.com/questions/30123094](https://stackoverflow.com/questions/30123094/how-to-get-only-odata-count-without-value#answer-30154251))

**Comment by 27pchrisl** [comment]
Hi @RrOoSsSsOo, good catch. Yep I agree with the assessment on StackOverflow, fixed in [v5.26.0](https://github.com/flat3/lodata/releases/tag/v5.26.0)

**Comment by RrOoSsSsOo** [comment]
Thanks!
